src/Elm/Parser/Declarations.elm

```diff
@@ -129,6 +126,78 @@ functionWithNameNode start ((Node _ startName) as startNameNode) maybeDocumentat
         ]
 
 
+letFunctionWithoutDocumentationWithSignatureWithNameAndMaybeLayoutBacktrackable : Parser State (Node Declaration)
+letFunctionWithoutDocumentationWithSignatureWithNameAndMaybeLayoutBacktrackable =
```

Comment:
That's a mouthful :sweat_smile: 
The name is fine, I'm happy for anyone to rename it later if they find a better name, but... why the `let` at the beginning? This is not for a `let..in` function, right?

Reply:
got renamed in a later commit

Reply:
I just saw time, I was going to edit my comment but you were too quick :sweat_smile: